### PR TITLE
Feat/v1.2 landing

### PR DESCRIPTION
**Pull request by mariomui**
This adds a test for my pictures.
Please ignore the handleImageload commit.

### CODE REVIEW

File: __test__/CarouselLanding.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const tree = shallow(<CarouselLanding />);
+    tree.setState({
+      imageUrls: [
+        'http://google.com',
+        'http://google.com',
+        'http://google.com',
+        'http://google.com',
+      ],
+    });
+    expect(tree.find(CarouselCard)).to.have.lengthOf(4);
```
Comment (wusihong) [inline __test__/CarouselLanding.test.js]: length of 4 or length of 5?

File: __test__/CarouselLanding.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Enzyme.configure({ adapter: new Adapter() });
+
+// describe('hey', () => {
+
+// });
+describe('Carousel Landing', () => {
+  // const tree = mount(<CarouselLanding />);
+
+  it('to render five carousel card components', () => {
```
Comment (wusihong) [inline __test__/CarouselLanding.test.js]: suggest editing for clarity as follows: 

it('renders five CarouselCard components')...

also: why do you call it carousel "card"? what is the context of using the word "card"?

File: __test__/CarouselLanding.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import Enzyme, { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { expect } from 'chai';
+import Adapter from 'enzyme-adapter-react-16';
+// import renderer from 'react-test-renderer';
+import { Z_TREES } from 'zlib';
+import CarouselCentral from '../client/CarouselCentral';
+import CarouselLanding from '../client/components/CarouselLanding';
+import CarouselCard from '../client/components/CarouselCard';
+
```
Comment (wusihong) [inline __test__/CarouselLanding.test.js]: lots of imports above, are you you sure you need all of them? maybe you are setting thins up for more tests...but at least based on what you have below, seems like some of the imports are unecessary

File: __test__/CarouselCentral.test.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import CarouselCentral from '../client/CarouselCentral';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+// describe('hey', () => {
+
+// });
+
+describe('Carousel Central', () => {
```
Comment (wusihong) [inline __test__/CarouselCentral.test.js]: change to 'CarouselCentral'  rather than 'Carousel Central'

File: __test__/CarouselLanding.test.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+Enzyme.configure({ adapter: new Adapter() });
+
+// describe('hey', () => {
+
+// });
+describe('Carousel Landing', () => {
```
Comment (wusihong) [inline __test__/CarouselLanding.test.js]: 'CarouselLanding'   no space between two words -- match to the actual component name

File: __test__/CarouselLanding.test.js (hunk 2 above)
Comment (mariomui) [reply __test__/CarouselLanding.test.js]: Each picture is a card component. The first Card has to be bigger than the other ones. The other 4 cards have to stacked in a quadrant.
Each Card has some information like it's the first card or the second card and what kind of room it is in. It's not just a picture. It's more like a Trading Card.

**Pull request merged**